osgar/tools/log2map.py

```diff
@@ -0,0 +1,50 @@
+"""
+  Create map (gridmap image)
+"""
+import cv2  # for video output
+import numpy as np  # faster depth data processing
+
+from osgar.logger import LogReader, lookup_stream_names, lookup_stream_id
+from osgar.lib.serialize import deserialize
+
+SCALE = 10  # 1 pixel is 1dm
+BORDER_PX = 10  # extra border
+
+
+def main():
+    import argparse
+
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('logfile', help='recorded log file')
+    parser.add_argument('--out', help='map output file', default='world.jpg')
+    parser.add_argument('--pose2d', help='stream ID for pose2d messages')
+    args = parser.parse_args()
+    if not any([args.pose2d,]):
```

Comment:
Is this equivalent 
```python
if args.pose2d == None:
```
? If so, why do it differently?

Reply:
OK, I will change it to `if args.pose2d is None:` ... it was copy & paste and I expected to mix lidar and depth data soon ... but it turned out, that even pose2d is sufficient (yeah, there was also pose3d for a moment). I will change it to simple if statement and extend it when needed. Thanks